sql/src/main/java/io/crate/planner/consumer/MultiSourceGroupByConsumer.java

```diff
@@ -106,10 +105,6 @@ private static void removePostGroupingActionsFromQuerySpec(MultiSourceSelect mss
             querySpec.outputs(outputs);
             querySpec.hasAggregates(false);
             removePostGroupingActions(querySpec);
-
```

Comment:
pls update the documentation of this method and describe a bit more what it does. maybe it would make sense to rename the method?